indico/htdocs/sass/partials/_tables.scss

```diff
@@ -30,6 +36,27 @@ table.i-table {
         font-weight: bold;
         text-align: left;
         background-color: #EAEAEA;
+
+        &:not(.sorter-false) {
```

Comment:
Is this class added by tablesorter? Otherwise you probably want `:not([data-sorter="false"])` here